engine/apps/api/views/alert_group.py

```diff
@@ -469,6 +470,11 @@ def enrich(self, alert_groups):
 
         return alert_groups
 
+    def destroy(self, request, *args, **kwargs):
+        instance = self.get_object()
+        delete_alert_group.apply_async((instance.pk, request.user.pk))
```

Comment:
follows similar logic to our `DELETE /alertgroups/<id>` [public API endpoint handler](https://github.com/grafana/oncall/blob/dev/engine/apps/public_api/views/incidents.py#L112) (don't think we need to implement the concept of "wipe" on the internal API endpoint for now)